enterprise/backend/test/metabase_enterprise/serialization/v2/e2e/yaml_test.clj

```diff
@@ -28,50 +28,56 @@
 (defn- strip-labels [path]
   (mapv #(dissoc % :label) path))
 
-(defn- random-key [prefix n]
-  (keyword (str prefix (rand-int n))))
+(defn- random-key
```

Comment:
This might be pedantic but I would name this `random-keyword` to make its purpose clearer. While all keywords are keys not all keys are keywords. I had to read the implementation to figure out what this function did -- maybe it was returning a string or something? A slight name tweak would have made it clear without having to read the code